Using MAPPING_STR=12121212, VERSION_DIR=v4
Running /platform/source/s01/v4/m12121212/m12121212_q10_t0.sh
Collecting pip
  Downloading pip-23.3.1-py3-none-any.whl (2.1 MB)
[?25l[K     |‚ñè                               | 10 kB 12.1 MB/s eta 0:00:01[K     |‚ñé                               | 20 kB 5.6 MB/s eta 0:00:01[K     |‚ñå                               | 30 kB 8.2 MB/s eta 0:00:01[K     |‚ñã                               | 40 kB 1.1 MB/s eta 0:00:02[K     |‚ñä                               | 51 kB 1.4 MB/s eta 0:00:02[K     |‚ñà                               | 61 kB 1.6 MB/s eta 0:00:02[K     |‚ñà                               | 71 kB 1.9 MB/s eta 0:00:02[K     |‚ñà‚ñé                              | 81 kB 2.1 MB/s eta 0:00:01[K     |‚ñà‚ñç                              | 92 kB 2.3 MB/s eta 0:00:01[K     |‚ñà‚ñå                              | 102 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñä                              | 112 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñâ                              | 122 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà                              | 133 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñè                             | 143 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñé                             | 153 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñå                             | 163 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñã                             | 174 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñâ                             | 184 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà                             | 194 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà                             | 204 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñé                            | 215 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñç                            | 225 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñã                            | 235 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñä                            | 245 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñâ                            | 256 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà                            | 266 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñè                           | 276 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñç                           | 286 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñå                           | 296 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñã                           | 307 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñâ                           | 317 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà                           | 327 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                          | 337 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                          | 348 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                          | 358 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                          | 368 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                          | 378 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                          | 389 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                          | 399 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                         | 409 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                         | 419 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                         | 430 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                         | 440 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                         | 450 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                         | 460 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                        | 471 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                        | 481 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                        | 491 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                        | 501 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                        | 512 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                        | 522 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                        | 532 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                       | 542 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                       | 552 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                       | 563 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                       | 573 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                       | 583 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                       | 593 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                      | 604 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                      | 614 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                      | 624 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                      | 634 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                      | 645 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                      | 655 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                      | 665 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                     | 675 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                     | 686 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                     | 696 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                     | 706 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                     | 716 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                     | 727 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                    | 737 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                    | 747 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                    | 757 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                    | 768 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                    | 778 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                    | 788 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                   | 798 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                   | 808 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                   | 819 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                   | 829 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                   | 839 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                   | 849 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                   | 860 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                  | 870 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                  | 880 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                  | 890 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                  | 901 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                  | 911 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                  | 921 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                 | 931 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                 | 942 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                 | 952 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                 | 962 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                 | 972 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                 | 983 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                 | 993 kB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                | 1.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                | 1.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                | 1.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                | 1.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                | 1.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè               | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé               | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå               | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã               | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ               | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà               | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà               | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé              | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç              | 1.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã              | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä              | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ              | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà              | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè             | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç             | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå             | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã             | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ             | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà             | 1.2 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè            | 1.3 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé            | 1.3 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç            | 1.3 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã            | 1.3 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä            | 1.3 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà            | 1.3 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà            | 1.3 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè           | 1.3 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç           | 1.3 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå           | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä           | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ           | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà           | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè          | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé          | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå          | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã          | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä          | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà          | 1.4 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà          | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé         | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç         | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå         | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä         | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ         | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà         | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè        | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé        | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå        | 1.5 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã        | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ        | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà        | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà        | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé       | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç       | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã       | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä       | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ       | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà       | 1.6 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè      | 1.7 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç      | 1.7 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå      | 1.7 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã      | 1.7 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ      | 1.7 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà      | 1.7 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè     | 1.7 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé     | 1.7 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç     | 1.7 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã     | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä     | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà     | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà     | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè    | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç    | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå    | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä    | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ    | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà    | 1.8 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè   | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé   | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå   | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã   | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä   | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà   | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà   | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé  | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç  | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå  | 1.9 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä  | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ  | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà  | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà | 2.0 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé| 2.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç| 2.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã| 2.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä| 2.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ| 2.1 MB 2.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 2.1 MB 2.6 MB/s 
[?25hInstalling collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 20.0.2
    Not uninstalling pip at /usr/lib/python3/dist-packages, outside environment /usr
    Can't uninstall 'pip'. No files were found to uninstall.
Successfully installed pip-23.3.1
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4/m12121212/m12121212_q11_t0.sh
Running /platform/source/s01/v4/m12121212/m12121212_q12_t0.sh
Running /platform/source/s01/v4/m12121212/m12121212_q13_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]            Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
0% [Waiting for headers]0% [Waiting for headers]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
Calculating upgrade... 0%Calculating upgrade... 50%Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3 set to manually installed.
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q14_t0.sh
/platform/source/s01/v4/m12121212/m12121212_q14_t0.sh: line 6: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q14_t0.sh: line 7: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0m[31mERROR: Could not find a version that satisfies the requirement direct-redis==0.0.0 (from versions: 0.1.0, 0.1.1, 0.1.2, 0.1.3, 0.1.4, 0.1.5, 0.1.6, 0.1.7, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9, 0.3.0, 0.3.1)[0m[31m
[0m[31mERROR: No matching distribution found for direct-redis==0.0.0[0m[31m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q15_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Requirement already satisfied: pip in /usr/local/lib/python3.8/dist-packages (23.3.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q16_t0.sh
/platform/source/s01/v4/m12121212/m12121212_q16_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q16_t0.sh: line 8: sudo: command not found
Requirement already satisfied: pip in /usr/local/lib/python3.8/dist-packages (23.3.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mCollecting pymongo==3.12.3
  Downloading pymongo-3.12.3-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (527 kB)
[?25l     [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m0.0/527.3 kB[0m [31m?[0m eta [36m-:--:--[0m[2K     [91m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m[90m‚ï∫[0m[90m‚îÅ‚îÅ[0m [32m491.5/527.3 kB[0m [31m14.9 MB/s[0m eta [36m0:00:01[0m[2K     [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m527.3/527.3 kB[0m [31m13.6 MB/s[0m eta [36m0:00:00[0m
[?25hInstalling collected packages: pymongo
  Attempting uninstall: pymongo
    Found existing installation: pymongo 4.6.0
    Uninstalling pymongo-4.6.0:
      Successfully uninstalled pymongo-4.6.0
Successfully installed pymongo-3.12.3
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q17_t0.sh
chmod: cannot access 'install_dependencies.sh': No such file or directory
Running /platform/source/s01/v4/m12121212/m12121212_q18_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
0% [Waiting for headers]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
The following additional packages will be installed:
  libboost-filesystem1.71.0 libboost-iostreams1.71.0
  libboost-program-options1.71.0 libgoogle-perftools4 libpcap0.8 libpcrecpp0v5
  libsnappy1v5 libstemmer0d libtcmalloc-minimal4 libyaml-cpp0.6 mongo-tools
  mongodb-clients mongodb-server mongodb-server-core
The following NEW packages will be installed:
  libboost-filesystem1.71.0 libboost-iostreams1.71.0
  libboost-program-options1.71.0 libgoogle-perftools4 libpcap0.8 libpcrecpp0v5
  libsnappy1v5 libstemmer0d libtcmalloc-minimal4 libyaml-cpp0.6 mongo-tools
  mongodb mongodb-clients mongodb-server mongodb-server-core
0 upgraded, 15 newly installed, 0 to remove and 0 not upgraded.
Need to get 49.3 MB of archives.
After this operation, 215 MB of additional disk space will be used.
0% [Working]            Get:1 http://ports.ubuntu.com/ubuntu-ports focal/main arm64 libpcap0.8 arm64 1.9.1-3 [120 kB]
0% [1 libpcap0.8 1195 B/120 kB 1%]                                  2% [Working]            Get:2 http://ports.ubuntu.com/ubuntu-ports focal/main arm64 libboost-filesystem1.71.0 arm64 1.71.0-6ubuntu6 [238 kB]
2% [2 libboost-filesystem1.71.0 5539 B/238 kB 2%]                                                 3% [Waiting for headers]                        Get:3 http://ports.ubuntu.com/ubuntu-ports focal/main arm64 libboost-iostreams1.71.0 arm64 1.71.0-6ubuntu6 [234 kB]
3% [3 libboost-iostreams1.71.0 3702 B/234 kB 2%]                                                5% [Waiting for headers]                        Get:4 http://ports.ubuntu.com/ubuntu-ports focal/main arm64 libboost-program-options1.71.0 arm64 1.71.0-6ubuntu6 [326 kB]
5% [4 libboost-program-options1.71.0 1867 B/326 kB 1%]                                                      7% [Waiting for headers]                        Get:5 http://ports.ubuntu.com/ubuntu-ports focal/main arm64 libtcmalloc-minimal4 arm64 2.7-1ubuntu2 [85.6 kB]
7% [5 libtcmalloc-minimal4 5448 B/85.6 kB 6%]                                             8% [Waiting for headers]                        Get:6 http://ports.ubuntu.com/ubuntu-ports focal/main arm64 libgoogle-perftools4 arm64 2.7-1ubuntu2 [178 kB]
8% [6 libgoogle-perftools4 11.3 kB/178 kB 6%]                                             10% [Waiting for headers]                         Get:7 http://ports.ubuntu.com/ubuntu-ports focal-updates/main arm64 libpcrecpp0v5 arm64 2:8.39-12ubuntu0.1 [14.7 kB]
10% [7 libpcrecpp0v5 7836 B/14.7 kB 53%]                                        11% [Waiting for headers]                         Get:8 http://ports.ubuntu.com/ubuntu-ports focal/main arm64 libstemmer0d arm64 0+svn585-2 [60.6 kB]
11% [8 libstemmer0d 0 B/60.6 kB 0%]                                   13% [Waiting for headers]                         Get:9 http://ports.ubuntu.com/ubuntu-ports focal/main arm64 libyaml-cpp0.6 arm64 0.6.2-4ubuntu1 [107 kB]
13% [9 libyaml-cpp0.6 1760 B/107 kB 2%]                                       14% [Waiting for headers]                         Get:10 http://ports.ubuntu.com/ubuntu-ports focal/universe arm64 mongo-tools arm64 3.6.3-0ubuntu1 [10.3 MB]
14% [10 mongo-tools 9880 B/10.3 MB 0%]                                      32% [Waiting for headers]                         Get:11 http://ports.ubuntu.com/ubuntu-ports focal/main arm64 libsnappy1v5 arm64 1.1.8-1build1 [16.0 kB]
32% [11 libsnappy1v5 16.0 kB/16.0 kB 100%]                                          34% [Working]             Get:12 http://ports.ubuntu.com/ubuntu-ports focal-updates/universe arm64 mongodb-clients arm64 1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3 [18.6 MB]
34% [12 mongodb-clients 9963 B/18.6 MB 0%]34% [12 mongodb-clients 366 kB/18.6 MB 2%]49% [12 mongodb-clients 9351 kB/18.6 MB 50%]                                            65% [Working]             Get:13 http://ports.ubuntu.com/ubuntu-ports focal-updates/universe arm64 mongodb-server-core arm64 1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3 [18.9 MB]
65% [13 mongodb-server-core 16.4 kB/18.9 MB 0%]77% [13 mongodb-server-core 6970 kB/18.9 MB 37%]92% [13 mongodb-server-core 16.4 MB/18.9 MB 87%]                                                97% [Waiting for headers]                         Get:14 http://ports.ubuntu.com/ubuntu-ports focal-updates/universe arm64 mongodb-server all 1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3 [12.5 kB]
97% [14 mongodb-server 12.5 kB/12.5 kB 100%]                                            99% [Working]             Get:15 http://ports.ubuntu.com/ubuntu-ports focal-updates/universe arm64 mongodb arm64 1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3 [9640 B]
99% [15 mongodb 9640 B/9640 B 100%]                                   100% [Working]              Fetched 49.3 MB in 5s (10.9 MB/s)
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package libpcap0.8:arm64.
(Reading database ... (Reading database ... 5%(Reading database ... 10%(Reading database ... 15%(Reading database ... 20%(Reading database ... 25%(Reading database ... 30%(Reading database ... 35%(Reading database ... 40%(Reading database ... 45%(Reading database ... 50%(Reading database ... 55%(Reading database ... 60%(Reading database ... 65%(Reading database ... 70%(Reading database ... 75%(Reading database ... 80%(Reading database ... 85%(Reading database ... 90%(Reading database ... 95%(Reading database ... 100%(Reading database ... 14446 files and directories currently installed.)
Preparing to unpack .../00-libpcap0.8_1.9.1-3_arm64.deb ...
Unpacking libpcap0.8:arm64 (1.9.1-3) ...
Selecting previously unselected package libboost-filesystem1.71.0:arm64.
Preparing to unpack .../01-libboost-filesystem1.71.0_1.71.0-6ubuntu6_arm64.deb ...
Unpacking libboost-filesystem1.71.0:arm64 (1.71.0-6ubuntu6) ...
Selecting previously unselected package libboost-iostreams1.71.0:arm64.
Preparing to unpack .../02-libboost-iostreams1.71.0_1.71.0-6ubuntu6_arm64.deb ...
Unpacking libboost-iostreams1.71.0:arm64 (1.71.0-6ubuntu6) ...
Selecting previously unselected package libboost-program-options1.71.0:arm64.
Preparing to unpack .../03-libboost-program-options1.71.0_1.71.0-6ubuntu6_arm64.deb ...
Unpacking libboost-program-options1.71.0:arm64 (1.71.0-6ubuntu6) ...
Selecting previously unselected package libtcmalloc-minimal4:arm64.
Preparing to unpack .../04-libtcmalloc-minimal4_2.7-1ubuntu2_arm64.deb ...
Unpacking libtcmalloc-minimal4:arm64 (2.7-1ubuntu2) ...
Selecting previously unselected package libgoogle-perftools4:arm64.
Preparing to unpack .../05-libgoogle-perftools4_2.7-1ubuntu2_arm64.deb ...
Unpacking libgoogle-perftools4:arm64 (2.7-1ubuntu2) ...
Selecting previously unselected package libpcrecpp0v5:arm64.
Preparing to unpack .../06-libpcrecpp0v5_2%3a8.39-12ubuntu0.1_arm64.deb ...
Unpacking libpcrecpp0v5:arm64 (2:8.39-12ubuntu0.1) ...
Selecting previously unselected package libstemmer0d:arm64.
Preparing to unpack .../07-libstemmer0d_0+svn585-2_arm64.deb ...
Unpacking libstemmer0d:arm64 (0+svn585-2) ...
Selecting previously unselected package libyaml-cpp0.6:arm64.
Preparing to unpack .../08-libyaml-cpp0.6_0.6.2-4ubuntu1_arm64.deb ...
Unpacking libyaml-cpp0.6:arm64 (0.6.2-4ubuntu1) ...
Selecting previously unselected package mongo-tools.
Preparing to unpack .../09-mongo-tools_3.6.3-0ubuntu1_arm64.deb ...
Unpacking mongo-tools (3.6.3-0ubuntu1) ...
Selecting previously unselected package libsnappy1v5:arm64.
Preparing to unpack .../10-libsnappy1v5_1.1.8-1build1_arm64.deb ...
Unpacking libsnappy1v5:arm64 (1.1.8-1build1) ...
Selecting previously unselected package mongodb-clients.
Preparing to unpack .../11-mongodb-clients_1%3a3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3_arm64.deb ...
Unpacking mongodb-clients (1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3) ...
Selecting previously unselected package mongodb-server-core.
Preparing to unpack .../12-mongodb-server-core_1%3a3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3_arm64.deb ...
Unpacking mongodb-server-core (1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3) ...
Selecting previously unselected package mongodb-server.
Preparing to unpack .../13-mongodb-server_1%3a3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3_all.deb ...
Unpacking mongodb-server (1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3) ...
Selecting previously unselected package mongodb.
Preparing to unpack .../14-mongodb_1%3a3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3_arm64.deb ...
Unpacking mongodb (1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3) ...
Setting up libpcrecpp0v5:arm64 (2:8.39-12ubuntu0.1) ...
Setting up libtcmalloc-minimal4:arm64 (2.7-1ubuntu2) ...
Setting up libboost-filesystem1.71.0:arm64 (1.71.0-6ubuntu6) ...
Setting up libboost-iostreams1.71.0:arm64 (1.71.0-6ubuntu6) ...
Setting up libsnappy1v5:arm64 (1.1.8-1build1) ...
Setting up libpcap0.8:arm64 (1.9.1-3) ...
Setting up libyaml-cpp0.6:arm64 (0.6.2-4ubuntu1) ...
Setting up mongo-tools (3.6.3-0ubuntu1) ...
Setting up libstemmer0d:arm64 (0+svn585-2) ...
Setting up libgoogle-perftools4:arm64 (2.7-1ubuntu2) ...
Setting up libboost-program-options1.71.0:arm64 (1.71.0-6ubuntu6) ...
Setting up mongodb-clients (1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3) ...
Setting up mongodb-server-core (1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3) ...
Setting up mongodb-server (1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3) ...
invoke-rc.d: could not determine current runlevel
invoke-rc.d: policy-rc.d denied execution of start.
Setting up mongodb (1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3) ...
Processing triggers for libc-bin (2.31-0ubuntu9.12) ...
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
The following additional packages will be installed:
  libhiredis0.14 libjemalloc2 liblua5.1-0 lua-bitop lua-cjson redis-tools
Suggested packages:
  ruby-redis
The following NEW packages will be installed:
  libhiredis0.14 libjemalloc2 liblua5.1-0 lua-bitop lua-cjson redis-server
  redis-tools
0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.
Need to get 812 kB of archives.
After this operation, 3741 kB of additional disk space will be used.
0% [Working]            Get:1 http://ports.ubuntu.com/ubuntu-ports focal/universe arm64 libhiredis0.14 arm64 0.14.0-6 [28.2 kB]
0% [1 libhiredis0.14 2645 B/28.2 kB 9%]                                       6% [Working]            Get:2 http://ports.ubuntu.com/ubuntu-ports focal/universe arm64 libjemalloc2 arm64 5.2.1-1ubuntu1 [172 kB]
6% [2 libjemalloc2 5539 B/172 kB 3%]                                    25% [Waiting for headers]                         Get:3 http://ports.ubuntu.com/ubuntu-ports focal/universe arm64 liblua5.1-0 arm64 5.1.5-8.1build4 [92.0 kB]
26% [3 liblua5.1-0 1371 B/92.0 kB 1%]                                     37% [Waiting for headers]                         Get:4 http://ports.ubuntu.com/ubuntu-ports focal/universe arm64 lua-bitop arm64 1.0.2-5 [6196 B]
38% [4 lua-bitop 4096 B/6196 B 66%]                                   41% [Waiting for headers]                         Get:5 http://ports.ubuntu.com/ubuntu-ports focal/universe arm64 lua-cjson arm64 2.1.0+dfsg-2.1 [15.6 kB]
41% [5 lua-cjson 545 B/15.6 kB 3%]                                  45% [Waiting for headers]                         Get:6 http://ports.ubuntu.com/ubuntu-ports focal-updates/universe arm64 redis-tools arm64 5:5.0.7-2ubuntu0.1 [461 kB]
45% [6 redis-tools 2032 B/461 kB 0%]                                    93% [Waiting for headers]                         Get:7 http://ports.ubuntu.com/ubuntu-ports focal-updates/universe arm64 redis-server arm64 5:5.0.7-2ubuntu0.1 [37.4 kB]
94% [7 redis-server 5473 B/37.4 kB 15%]                                       100% [Working]              Fetched 812 kB in 1s (943 kB/s)
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package libhiredis0.14:arm64.
(Reading database ... (Reading database ... 5%(Reading database ... 10%(Reading database ... 15%(Reading database ... 20%(Reading database ... 25%(Reading database ... 30%(Reading database ... 35%(Reading database ... 40%(Reading database ... 45%(Reading database ... 50%(Reading database ... 55%(Reading database ... 60%(Reading database ... 65%(Reading database ... 70%(Reading database ... 75%(Reading database ... 80%(Reading database ... 85%(Reading database ... 90%(Reading database ... 95%(Reading database ... 100%(Reading database ... 14570 files and directories currently installed.)
Preparing to unpack .../0-libhiredis0.14_0.14.0-6_arm64.deb ...
Unpacking libhiredis0.14:arm64 (0.14.0-6) ...
Selecting previously unselected package libjemalloc2:arm64.
Preparing to unpack .../1-libjemalloc2_5.2.1-1ubuntu1_arm64.deb ...
Unpacking libjemalloc2:arm64 (5.2.1-1ubuntu1) ...
Selecting previously unselected package liblua5.1-0:arm64.
Preparing to unpack .../2-liblua5.1-0_5.1.5-8.1build4_arm64.deb ...
Unpacking liblua5.1-0:arm64 (5.1.5-8.1build4) ...
Selecting previously unselected package lua-bitop:arm64.
Preparing to unpack .../3-lua-bitop_1.0.2-5_arm64.deb ...
Unpacking lua-bitop:arm64 (1.0.2-5) ...
Selecting previously unselected package lua-cjson:arm64.
Preparing to unpack .../4-lua-cjson_2.1.0+dfsg-2.1_arm64.deb ...
Unpacking lua-cjson:arm64 (2.1.0+dfsg-2.1) ...
Selecting previously unselected package redis-tools.
Preparing to unpack .../5-redis-tools_5%3a5.0.7-2ubuntu0.1_arm64.deb ...
Unpacking redis-tools (5:5.0.7-2ubuntu0.1) ...
Selecting previously unselected package redis-server.
Preparing to unpack .../6-redis-server_5%3a5.0.7-2ubuntu0.1_arm64.deb ...
Unpacking redis-server (5:5.0.7-2ubuntu0.1) ...
Setting up libjemalloc2:arm64 (5.2.1-1ubuntu1) ...
Setting up lua-cjson:arm64 (2.1.0+dfsg-2.1) ...
Setting up lua-bitop:arm64 (1.0.2-5) ...
Setting up liblua5.1-0:arm64 (5.1.5-8.1build4) ...
Setting up libhiredis0.14:arm64 (0.14.0-6) ...
Setting up redis-tools (5:5.0.7-2ubuntu0.1) ...
Setting up redis-server (5:5.0.7-2ubuntu0.1) ...
invoke-rc.d: could not determine current runlevel
invoke-rc.d: policy-rc.d denied execution of start.
Processing triggers for libc-bin (2.31-0ubuntu9.12) ...
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (3.12.3)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q19_t0.sh
/platform/source/s01/v4/m12121212/m12121212_q19_t0.sh: line 6: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q19_t0.sh: line 9: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q19_t0.sh: line 12: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q19_t0.sh: line 17: sudo: command not found
Running /platform/source/s01/v4/m12121212/m12121212_q1_t0.sh
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt install python3.8-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.

Failing command: ['/venv_query_redis/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']

/platform/source/s01/v4/m12121212/m12121212_q1_t0.sh: line 8: venv_query_redis/bin/activate: No such file or directory
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q20_t0.sh
/platform/source/s01/v4/m12121212/m12121212_q20_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q20_t0.sh: line 5: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (3.12.3)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q21_t0.sh
Running /platform/source/s01/v4/m12121212/m12121212_q22_t0.sh
Running /platform/source/s01/v4/m12121212/m12121212_q2_t0.sh
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (3.12.3)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q3_t0.sh
/platform/source/s01/v4/m12121212/m12121212_q3_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q3_t0.sh: line 6: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (3.12.3)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q4_t0.sh
/platform/source/s01/v4/m12121212/m12121212_q4_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q4_t0.sh: line 5: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (3.12.3)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q5_t0.sh
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (3.12.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q6_t0.sh
/platform/source/s01/v4/m12121212/m12121212_q6_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q6_t0.sh: line 5: sudo: command not found
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q7_t0.sh
/platform/source/s01/v4/m12121212/m12121212_q7_t0.sh: line 6: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q7_t0.sh: line 9: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (3.12.3)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q8_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
0% [Waiting for headers]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (3.12.3)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Collecting datetime
  Downloading DateTime-5.3-py3-none-any.whl.metadata (33 kB)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Collecting zope.interface (from datetime)
  Downloading zope.interface-6.1-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (41 kB)
[?25l     [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m0.0/41.7 kB[0m [31m?[0m eta [36m-:--:--[0m[2K     [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m41.7/41.7 kB[0m [31m7.8 MB/s[0m eta [36m0:00:00[0m
[?25hRequirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Requirement already satisfied: setuptools in /usr/lib/python3/dist-packages (from zope.interface->datetime) (45.2.0)
Downloading DateTime-5.3-py3-none-any.whl (52 kB)
[?25l   [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m0.0/52.2 kB[0m [31m?[0m eta [36m-:--:--[0m[2K   [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m52.2/52.2 kB[0m [31m10.3 MB/s[0m eta [36m0:00:00[0m
[?25hDownloading zope.interface-6.1-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (249 kB)
[?25l   [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m0.0/249.6 kB[0m [31m?[0m eta [36m-:--:--[0m[2K   [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m249.6/249.6 kB[0m [31m17.2 MB/s[0m eta [36m0:00:00[0m
[?25hInstalling collected packages: zope.interface, datetime
Successfully installed datetime-5.3 zope.interface-6.1
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m12121212/m12121212_q9_t0.sh
/platform/source/s01/v4/m12121212/m12121212_q9_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m12121212/m12121212_q9_t0.sh: line 7: sudo: command not found
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (3.12.3)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mpython /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q10_t0.py /platform/output/s01/v4/m12121212/m12121212_q10_t0.csv /platform/expected/s01/q10.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q10_t0.py", line 28, in <module>
    merged_df = customer_df.merge(orders_df, left_on='C_CUSTKEY', right_on='O_CUSTKEY')
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 9843, in merge
    return merge(
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 148, in merge
    op = _MergeOperation(
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 737, in __init__
    ) = self._get_merge_keys()
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 1203, in _get_merge_keys
    right_keys.append(right._get_label_or_level_values(rk))
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py", line 1778, in _get_label_or_level_values
    raise KeyError(key)
KeyError: 'O_CUSTKEY'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q11_t0.py /platform/output/s01/v4/m12121212/m12121212_q11_t0.csv /platform/expected/s01/q11.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q11_t0.py", line 19, in <module>
    supplier = pd.read_pickle(supplier_df_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/pickle.py", line 179, in read_pickle
    with get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 716, in get_handle
    ioargs = _get_filepath_or_buffer(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 456, in _get_filepath_or_buffer
    raise ValueError(msg)
ValueError: Invalid file path or buffer object type: <class 'pandas.core.frame.DataFrame'>
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q12_t0.py /platform/output/s01/v4/m12121212/m12121212_q12_t0.csv /platform/expected/s01/q12.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q12_t0.py", line 39, in <module>
    (lineitem_df['L_RECEIPTDATE'] >= pd.Timestamp('1994-01-01')) &
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/ops/common.py", line 81, in new_method
    return method(self, other)
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/arraylike.py", line 60, in __ge__
    return self._cmp_method(other, operator.ge)
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/series.py", line 6096, in _cmp_method
    res_values = ops.comparison_op(lvalues, rvalues, op)
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/ops/array_ops.py", line 279, in comparison_op
    res_values = op(lvalues, rvalues)
TypeError: '>=' not supported between instances of 'numpy.ndarray' and 'Timestamp'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q13_t0.py /platform/output/s01/v4/m12121212/m12121212_q13_t0.csv /platform/expected/s01/q13.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q13_t0.py", line 59, in <module>
    main()
  File "/platform/source/s01/v4/m12121212/m12121212_q13_t0.py", line 53, in main
    result_df = run_aggregation(mongo_db, redis_db)
  File "/platform/source/s01/v4/m12121212/m12121212_q13_t0.py", line 31, in run_aggregation
    customer = pd.read_json(customer_data, orient='records')
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q14_t0.py /platform/output/s01/v4/m12121212/m12121212_q14_t0.csv /platform/expected/s01/q14.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q14_t0.py", line 19, in <module>
    lineitem_df = pd.read_json(lineitem_df)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q15_t0.py /platform/output/s01/v4/m12121212/m12121212_q15_t0.csv /platform/expected/s01/q15.csv
/platform/source/s01/v4/m12121212/m12121212_q15_t0.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  final_result.sort_values(by='S_SUPPKEY', inplace=True)
>>> Expected ((1, 5))
               S_ADDRESS              S_NAME          S_PHONE  S_SUPPKEY  TOTAL_REVENUE
0  8mhrffG7D2WJBSQbOGstQ  Supplier#000000677  23-290-639-3315        677   1.614410e+06

>>> Actual ((1, 5))
               S_ADDRESS              S_NAME          S_PHONE  S_SUPPKEY  TOTAL_REVENUE
0  8mhrffG7D2WJBSQbOGstQ  Supplier#000000677  23-290-639-3315        677   1.614410e+06

Head row similarity: 1.0
Score.success_approx
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q16_t0.py /platform/output/s01/v4/m12121212/m12121212_q16_t0.csv /platform/expected/s01/q16.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q16_t0.py", line 28, in <module>
    supplier_df = pd.read_json(supplier_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q17_t0.py /platform/output/s01/v4/m12121212/m12121212_q17_t0.csv /platform/expected/s01/q17.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q17_t0.py", line 23, in <module>
    lineitem_df = pd.read_json(r.get('lineitem'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q18_t0.py /platform/output/s01/v4/m12121212/m12121212_q18_t0.csv /platform/expected/s01/q18.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q18_t0.py", line 18, in <module>
    customer_df = pd.read_json(StringIO(redis_client.get('customer')))
TypeError: initial_value must be str or None, not DataFrame
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q19_t0.py /platform/output/s01/v4/m12121212/m12121212_q19_t0.csv /platform/expected/s01/q19.csv
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/base.py", line 3653, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 147, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 176, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 7080, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 7088, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'L_SHIPMODE'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q19_t0.py", line 52, in <module>
    lineitem_df['L_SHIPMODE'].isin(['AIR', 'AIR REG']) &
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 3761, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/base.py", line 3655, in get_loc
    raise KeyError(key) from err
KeyError: 'L_SHIPMODE'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q1_t0.py /platform/output/s01/v4/m12121212/m12121212_q1_t0.csv /platform/expected/s01/q1.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q1_t0.py", line 44, in <module>
    query_redis()
  File "/platform/source/s01/v4/m12121212/m12121212_q1_t0.py", line 16, in query_redis
    df_lineitem = pd.read_json(lineitem_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q20_t0.py /platform/output/s01/v4/m12121212/m12121212_q20_t0.csv /platform/expected/s01/q20.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q20_t0.py", line 26, in <module>
    supplier_df = pd.read_json(redis_conn.get('supplier'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q21_t0.py /platform/output/s01/v4/m12121212/m12121212_q21_t0.csv /platform/expected/s01/q21.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q21_t0.py", line 68, in <module>
    main()
  File "/platform/source/s01/v4/m12121212/m12121212_q21_t0.py", line 62, in main
    supplier, lineitem = fetch_redis_data(redis_client)
  File "/platform/source/s01/v4/m12121212/m12121212_q21_t0.py", line 23, in fetch_redis_data
    supplier = pd.read_json(r.get('supplier'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q22_t0.py /platform/output/s01/v4/m12121212/m12121212_q22_t0.csv /platform/expected/s01/q22.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q22_t0.py", line 19, in <module>
    customers = pd.read_json(redis_client.get('customer'))
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 1579, in get
    return self.execute_command('GET', name)
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 878, in execute_command
    return self.parse_response(conn, command_name, **options)
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 892, in parse_response
    response = connection.read_response()
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 734, in read_response
    response = self._parser.read_response()
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 358, in read_response
    response = self.encoder.decode(response)
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 129, in decode
    value = value.decode(self.encoding, self.encoding_errors)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q2_t0.py /platform/output/s01/v4/m12121212/m12121212_q2_t0.csv /platform/expected/s01/q2.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q2_t0.py", line 48, in <module>
    main()
  File "/platform/source/s01/v4/m12121212/m12121212_q2_t0.py", line 41, in main
    region_df, supplier_df = get_redis_data(redis_connection)
  File "/platform/source/s01/v4/m12121212/m12121212_q2_t0.py", line 14, in get_redis_data
    region_df = pd.read_json(redis_connection.get('region'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q3_t0.py /platform/output/s01/v4/m12121212/m12121212_q3_t0.csv /platform/expected/s01/q3.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q3_t0.py", line 30, in <module>
    lineitem_df = pd.read_json(redis_client.get('lineitem'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q4_t0.py /platform/output/s01/v4/m12121212/m12121212_q4_t0.csv /platform/expected/s01/q4.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q4_t0.py", line 21, in <module>
    lineitem_df = pd.read_json(redis_client.get('lineitem'), orient='records')
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q5_t0.py /platform/output/s01/v4/m12121212/m12121212_q5_t0.csv /platform/expected/s01/q5.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q5_t0.py", line 17, in <module>
    region_df = pd.read_json(r.get('region'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q6_t0.py /platform/output/s01/v4/m12121212/m12121212_q6_t0.csv /platform/expected/s01/q6.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q6_t0.py", line 32, in <module>
    query_and_save_to_csv()
  File "/platform/source/s01/v4/m12121212/m12121212_q6_t0.py", line 10, in query_and_save_to_csv
    lineitem_df = pd.read_json(lineitem_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q7_t0.py /platform/output/s01/v4/m12121212/m12121212_q7_t0.csv /platform/expected/s01/q7.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q7_t0.py", line 51, in <module>
    supplier_df = fetch_redis_data(redis_conn, 'supplier')
  File "/platform/source/s01/v4/m12121212/m12121212_q7_t0.py", line 27, in fetch_redis_data
    df = pd.read_json(data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q8_t0.py /platform/output/s01/v4/m12121212/m12121212_q8_t0.csv /platform/expected/s01/q8.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q8_t0.py", line 20, in <module>
    supplier_df = pd.read_msgpack(redis_client.get('supplier'))
AttributeError: module 'pandas' has no attribute 'read_msgpack'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m12121212/m12121212_q9_t0.py /platform/output/s01/v4/m12121212/m12121212_q9_t0.csv /platform/expected/s01/q9.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m12121212/m12121212_q9_t0.py", line 20, in <module>
    supplier = pd.read_json(redis_conn.get('supplier'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
